packages/utils/src/crypto.ts

```diff
@@ -134,17 +134,8 @@ export function validateDecoding(
 }
 
 export function validateEncoding(opts?: CryptoTypes.EncodeOptions): CryptoTypes.EncodingValidation {
-  const type = opts?.type || TYPE_0;
```

Comment:
How come we're removing these checks? I think we should keep them since:

* there could be cases where `validateEncoding` is called without doing the manual message type check beforehand like we are here.
* If it's `TYPE_1` we should still validate that `senderPublicKey/receiverPublicKey` are present

Reply:
Good points. Thanks for bringing it up! Thought the validation would be handled by the external SDKs but makes sense to preserve it there